src/ui/public/filter_bar/lib/map_script.js

```diff
@@ -14,8 +14,7 @@ define(function () {
           if (filter.meta.formattedValue) {
             value = filter.meta.formattedValue;
           } else {
-            value = filter.script.script.params.value;
-            value = field.format.convert(value);
+            value = '' + filter.script.script.params.value;
```

Comment:
whew, good catch!  Reverting the label changes - will tackle that in a separate PR.